src/hooks/use-format-localized-timestamps.tsx

```diff
@@ -1,32 +1,36 @@
 import { Duration, intervalToDuration } from 'date-fns';
 import { useCallback } from 'react';
-import { formatDistance } from 'src/lib/date';
+import { formatDistance, formatDistanceShort, TimeFormat } from 'src/lib/date';
 
-function formatTimeDistance( duration: Duration ): string {
+function formatTimeDistance( duration: Duration, format: TimeFormat = 'long' ): string {
+	const fn = format === 'short' ? formatDistanceShort : formatDistance;
 	if ( duration.days && duration.days > 0 ) {
-		return formatDistance( 'xDays', duration.days );
+		return fn( 'xDays', duration.days );
 	} else if ( duration.hours && duration.hours > 0 ) {
-		return formatDistance( 'xHours', duration.hours );
+		return fn( 'xHours', duration.hours );
 	} else if ( duration.minutes && duration.minutes > 0 ) {
-		return formatDistance( 'xMinutes', duration.minutes );
+		return fn( 'xMinutes', duration.minutes );
 	} else {
-		return formatDistance( 'xMinutes', 1 );
+		return fn( 'xMinutes', 1 );
 	}
 }
 
 export function useFormatLocalizedTimestamps() {
-	const formatRelativeTime = useCallback( ( timestamp: string | null ): string => {
-		if ( timestamp === null ) {
-			return '';
-		}
+	const formatRelativeTime = useCallback(
+		( timestamp: string | null, format: TimeFormat = 'long' ): string => {
+			if ( timestamp === null ) {
+				return '';
+			}
 
-		const duration = intervalToDuration( {
-			start: new Date( timestamp ),
-			end: new Date(),
-		} );
+			const duration = intervalToDuration( {
+				start: new Date( timestamp ),
+				end: new Date(),
+			} );
 
-		return formatTimeDistance( duration );
-	}, [] );
+			return formatTimeDistance( duration, format );
+		},
+		[]
+	);
```

Comment:
Would it work to use `formatDistanceToNow` from `date-fns` here? See [docs](https://date-fns.org/v4.1.0/docs/formatDistanceToNow#). Seems like a potential simplification

Reply:
Thank you, Fredrik! Yeah, I think this is something we could consider in the future. I am not that familiar with `date-fns` yet, but looks like we could definitely make some improvements and simplifications in the way we calculate relative time in Studio.

Since we decided on a more straightforward solution to address the issue at hand, I did not explore `formatDistanceToNow` further today. I have created a dedicated task: https://github.com/Automattic/dotcom-forge/issues/10449 though.